app/code/Magento/Captcha/Test/Unit/Model/Customer/Plugin/AjaxLoginTest.php

```diff
@@ -79,31 +84,41 @@ protected function setUp()
         $this->captchaHelperMock->expects($this->once())->method('getCaptcha')
             ->with('user_login')->will($this->returnValue($this->captchaMock));
         $this->formIds = ['user_login'];
+        $this->serializerMock = $this->getMock(
+            \Magento\Framework\Serialize\SerializerInterface::class,
```

Comment:
Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`.